### PR TITLE
Feature/aqi 4632

### PR DESCRIPTION
**Pull request by MatthewDeRose-AI**

### CODE REVIEW

File: proto/one/enterprise/report/report_run.proto [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    // The name of the report output file
+    google.protobuf.StringValue reportOutputName = 7;
+    
+    // Start time of the report
+    JsonTicksDateTime startDate = 8;
+    
+    // End time of the report
+    JsonTicksDateTime endDate = 9;
+    
+    string createdById = 10;
```
Comment (AQI-DanG) [inline proto/one/enterprise/report/report_run.proto:39]: We have a RecordAuditInfo model that contains these two fields as well as modifiedById and modifiedOn.  Typically when using that we put it at the end of a model with the id 99 so if the model changes it is always at the end

**Review by AQI-DanG — CHANGES_REQUESTED** [review]
Couple quick changes and this should be fine

File: proto/one/enterprise/report/report_run.proto (hunk 1 above)
Comment (MatthewDeRose-AI) [reply proto/one/enterprise/report/report_run.proto:39]: We don't have ModifiedOn  or ModifiedBy for the ReportDefinitionRun table so I didn't think I would be able to include RecordAuditInfo.

**Review by tomsmith-ai — APPROVED** [review]

File: proto/one/enterprise/report/report_run.proto (hunk 1 above)
Comment (AQI-DanG) [reply proto/one/enterprise/report/report_run.proto:37]: You could still use it and just ignore those fields in mapping the protobuf to your database model, or you could add those columns to your sql table if desired, or you could leave it as is.

**Review by AQI-DanG — APPROVED** [review]

File: proto/one/enterprise/report/report_run.proto (hunk 1 above)
Comment (MatthewDeRose-AI) [reply proto/one/enterprise/report/report_run.proto:37]: Maybe I'll leave it for now and speak with Steve when he returns. I've sent him a message about the lack of the ModifiedBy/ModifiedOn fields so I just want to hear what his thoughts are

Comment (tomsmith-ai) [reply proto/one/enterprise/report/report_run.proto:37]: @MatthewDeRose-AI maybe just create a tech debt item describing adding audit capability to the report run table and if it is deemed vital we bring it in later.

**Pull request merged**